stories/Welcome.mdx

```diff
@@ -1,7 +1,138 @@
-import './styles.css';
+import './welcome-styles.css';
+import gridironLogo from './assets/gridironSurvivorLogo.svg';
+import appwriteLogo from './assets/appwriteLogo.svg';
+import blogRecorderLogo from './assets/blogRecorderLogo.svg';
+import frontendMentorLogo from './assets/frontendMentorLogo.svg';
+import gitkrakenLogo from './assets/gitkrakenLogo.svg';
+import pastelLogo from './assets/pastelLogo.svg';
+import Image from 'next/image';
 
-<h1>Welcome</h1>
+<html>
+  <body class="dark bg-background text-muted-foreground max-w-screen-xl mx-auto text-pretty flex flex-col space-y-36 pb-36">
+    <header class="header flex justify-center align-middle">
+      <a href="https://www.gridironsurvivor.com" target="_blank" rel="noreferrer" class="w-2/6 h-fit">
+        <Image class="w-full" src={gridironLogo} alt='Gridiron Survivor Football Logo' />
+      </a>
+    </header>
+    <section class="hero space-y-12">
+      <h1 class="pl-16">What is Gridiron Survivor?</h1>
+      <p class="text-xl text-pretty max-w-screen-lg mx-auto">
```

Comment:
Text feels a bit small from the screenshot, especially when there's that much text.

Reply:
I agree based on the screenshot. I had been building this in a 1080p window, but then took the screenshots on a 1440p display.

This is another spot where responsiveness will have to be built in.

Reply:
Increased font size.